Edge10.CouchDb.Client/KeyParameter.cs

```diff
@@ -11,12 +11,15 @@ public class KeyParameter
 
 		public static implicit operator KeyParameter(string key)
 		{
+			if (key == null)
```

Comment:
it doesn't handle this, result string will be `"?include_docs=true&key=null&custom=123"`